charts/opensearch-dashboards/README.md

```diff
@@ -51,6 +51,7 @@
 | `image.repository`                 | The OpenSearch Docker image                                                                                                                                                                                                                            | `opensearchproject/opensearch` |
 | `ingress`                          | Configurable [ingress][] to expose the OpenSearch service. See [values.yaml][] for an example                                                                                                                                                          | see [values.yaml][]                             |
 | `labels`                           | Configurable [labels][] applied to all OpenSearch pods                                                                                                                                                                                                 | `{}`                                            |
+| `lifecycle`                        | Allows you to add [lifecycle hooks](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/). See [values.yaml][] for an example                                                                                                     | `{}`                                            |
```

Comment:
@peterzhuamazon Thanks for adding this, can you please change this to See `values.yaml` for an example`